src/pages/ViewStudents.js

```diff
@@ -1,127 +1,141 @@
-            <input type="checkbox" name="Fifth" value="Fifth"></input>
-            <label for="Fifth">5th</label>
-            <input type="checkbox" name="Sixth" value="Sixth"></input>
-            <label for="Sixth">6th</label>
-            <input type="checkbox" name="Seventh" value="Seventh"></input>
-            <label for="Seventh">7th</label>
-            <input type="checkbox" name="Eight" value="Eight"></input>
-            <label for="Eighth">8th</label>
-            <input type="checkbox" name="Ninth" value="Ninth"></input>
-            <label for="Ninth">9th</label>
-            <input type="checkbox" name="Tenth" value="Tenth"></input>
-            <label for="Tenth">10th</label>
-            <input type="checkbox" name="Eleventh" valuediv="Eleventh"></input>
-            <label for="Eleventh">11th</label>
-            <input type="checkbox" name="Twelfth" value="Twelfth"></input>
-            <label for="Twelfth">12th</label>
-        </div>
-    </div>
-    <div class="TimezoneSection">Timezone</div>
-        {/* <br></br> */}
-        <div class="Dropdowndiv">
-            <input type="checkbox" name="EST" value="EST"></input>
-            <label for="EST">EST</label>
-            <input type="checkbox" name="PST" value="PST"></input>
-            <label for="PST">PST</label>
-            <input type="checkbox" name="CST" value="CST"></input>
-            <label for="CST">CST</label>
-            <input type="checkbox" name="MT" value="MT"></input>
-            <label for="MT">MT</label>
         </div>
-    </div>
-</div>
-
-        {/* <select class="dropdown">
-            <option disabled selected> Sort Options</option>
-        </select> */}
-    </>
-    
+    );
 }
 
-function TutorRow({ name, status, numStudents, maxStudents, subjects }) {
-    return <tr>
-        <th>{name}</th>
-        <td>{status}</td>
-        <td>{numStudents}</td>
-        <td>{maxStudents}</td>
-        <td>{subjects}</td>
-    </tr>
+function StudentRow({ name, status, tutor, subjects }) {
+    return (
+        <tr>
+            <td>{name}</td>
+            <td>{status}</td>
+            <td>{tutor}</td>
+            <td>{subjects}</td>
+        </tr>
+    );
 }
 
 function ViewStudents() {
-    return <>
-        <SortOptions />
+    const [filters, setFilters] = useState({
+        status: {},
+        subjects: {},
+        grades: {},
+        timezones: {}
+    });
 
-        <table class="viewtable">
-            <tr>
-                <th>Name</th>
-                <th>Status</th>
-                <th># of Students</th>
-                <th>Max Students</th>
-                <th>Subjects</th>
-            </tr>
-            <TutorRow name="John Doe" status="Matching" numStudents={0} maxStudents={15} subjects={"Math"} />
-        </table>
-    </>
+    const students = [
+        { name: "John Doe", status: "Matching in Progress", tutor: "Mr. Smith", subjects: "Math", grade: "First", timezone: "EST" },
```

Comment:
Grade level is stored as a number (0=K, 1=1st, ..., 13=post-secondary)

Reply:
Status is also either:
- `currentlyTutoring`
- `matchingInProgress`
- `unmatched`
- `updateNeeded`